README-dev.rst

```diff
@@ -71,36 +31,32 @@ Running Unit Tests
 ------------------
 Unit tests can be run like so::
 
-    python -m pytest --import-mode append tests/unit -k 'not (test_connection_initialization or test_cloud)'
-    EVENT_LOOP_MANAGER=gevent python -m pytest --import-mode append tests/unit/io/test_geventreactor.py
-    EVENT_LOOP_MANAGER=eventlet python -m pytest --import-mode append tests/unit/io/test_eventletreactor.py
+    python -m pytest tests/unit
+    EVENT_LOOP_MANAGER=gevent python -m pytest tests/unit/io/test_geventreactor.py
+    EVENT_LOOP_MANAGER=eventlet python -m pytest tests/unit/io/test_eventletreactor.py
 
 You can run a specific test method like so::
 
     python -m pytest tests/unit/test_connection.py::ConnectionTest::test_bad_protocol_version
 
 Running Integration Tests
 -------------------------
-In order to run integration tests, you must specify a version to run using the ``CASSANDRA_VERSION`` or ``DSE_VERSION`` environment variable::
-
-    CASSANDRA_VERSION=2.0.9 nosetests -w tests/integration/standard
-
-Or you can specify a cassandra directory (to test unreleased versions)::
+In order to run integration tests, you must specify a version to run using either of:
+* ``SCYLLA_VERSION`` (+ ``MAPPED_SCYLLA_VERSION=3.11.4``) - e.g. ``release:5.1``
```

Comment:
You can drop `MAPPED_SCYLLA_VERSION`, since https://github.com/scylladb/python-driver/pull/372 was merged